portfolio/src/main/webapp/Controller.js

```diff
@@ -0,0 +1,91 @@
+import CommentHandler from '/CommentHandler.js';
+import UserHandler from '/UserHandler.js';
+
+/**
+ * The adapter class which delegates actions required from view to 
+ * different service modules.
+ */
+export default class Controller {
+
+  constructor(document) {
+    /** 
+     * @private @type {string} the string of current user's username 
+     */
+    this.username_ = undefined;
```

Comment:
Mark member variables that don't change with @const annotations

Reply:
Sorry I missed these fields. Thanks